docker-compose.yml

```diff
@@ -125,6 +125,7 @@ services:
     environment:
       - NUXT_HOST=0.0.0.0
       - NUXT_PORT=3003
+      - NITRO_PORT=3003
```

Comment:
I don't think this is needed. NUXT_PORT is for the dev server, NITRO_PORT for the production server. Did you run into problems during local development?

Reply:
The e2e Test use the Produktion Preview, that needs the variable

Reply:
Ah I see, thanks